make.bat

```diff
@@ -0,0 +1,45 @@
+@echo off
+if /i "%1"=="docker-build" (
```

Comment:
j'ai dû ajouter cette ligne pour charger les variables du fichier .env.dev
`FOR /F "delims=" %%A IN (.env.dev) DO SET "%%A" `

Reply:
Effectivement, un mauvais copier/coller me l'a supprimé ;-)
Merci Matthieu ;-)

Reply:
![image](https://user-images.githubusercontent.com/100282361/215979966-7ef15b30-2cba-4025-8cc0-8705e1bb2125.png)

Reply:
Du coup, j'ai ajouté "call"
Sans cela c'est KO depuis ma VM windows...
J'espère que cela  n'aura pas d'impacte de ton côté

Reply:
non, c'est bon pour moi avec cette config.
Merci